lib/collectionspace/converter/publicart/acquisition.rb

```diff
@@ -6,31 +6,34 @@ class PublicArtAcquisition < CoreAcquisition
         def redefined_fields
           @redefined.concat([
             # not in publicart
-            'transferOfTitleNumber',
-            'groupPurchasePriceCurrency',
-            'groupPurchasePriceValue',
-            'objectOfferPriceCurrency',
-            'objectOfferPriceValue',
-            'objectPurchaseOfferPriceCurrency',
-            'objectPurchaseOfferPriceValue',
-            'objectPurchasePriceCurrency',
-            'objectPurchasePriceValue',
-            'originalObjectPurchasePriceCurrency',
-            'originalObjectPurchasePriceValue',
-            'approvalGroup',
-            'approvalIndividual',
-            'approvalStatus',
-            'approvalDate',
-            'approvalNote',
-            'acquisitionProvisos',
-            'fieldCollectionEventName',
-            'accessionDateGroup',
-            'acquisitionDateGroup',
+            'transferoftitlenumber',
```

Comment:
@Paulmulonzia - The fields here need to be all lowercase to match the attributes keys being passed around by the tool.

Reply:
@kspurgin, do they need to match field names in the csv document/sample data?. e.g. we might have the field **owner** which on the csv has both **ownerperson** and **ownerorganization** being matched to it.

Reply:
@Paulmulonzia -- yes, this list is really a list of the columns whose handling is being redefined or removed by a module. That's a good distinction to make.

Reply:
okay, noted.